src/net/ossl.cc

```diff
@@ -837,11 +841,25 @@ class session : public enable_shared_from_this<session>, public session_impl {
         assert(_output_pending.available());
     }
 
+    friend std::ostream & operator<<(std::ostream &os, const session & session) {
+        fmt::print(os, "{}:{}:{} -",
+                   session.get_type_string(),
+                   session.local_address(),
+                   session.remote_address());
+        return os;
```

Comment:
as explained above, i'd prefer avoid adding more `operator<<` overloads unless they are actually necessary and used.